Makefile

```diff
@@ -189,11 +189,8 @@ test-doc:
 	@ ./scripts/gear.sh test docs
 
 .PHONY: test-gear
-test-gear: # Crates except gclient, gcli, gsdk are excluded to significantly decrease time.
```

Comment:
why?

Reply:
these three packages are nested in `test_workspace`, so imo we just need to clarify for the following packages in the command so we can remove the annotation for these three packages here? or what about 

```
The following crates are excluded to significantly decrease time.
```